pyproject.toml

```diff
@@ -1,9 +1,7 @@
-[build-system]
-requires = ["setuptools", "setuptools-scm"]
 build-backend = "setuptools.build_meta"
 
 
-[tool.poetry]
```

Comment:
whoopsie, this [seems to have broken the nix stuff](https://github.com/pazz/alot/actions/runs/10181338695/job/28161237502#step:4:34)... 
The idea was to remove all poetry related configs and keep this file vanilla setuptools.
Would it suffice to adjust the nix script to reflect the namespace change?

I'll stop interfering with your branch now, sorry :)